client/signup/steps/rewind-form/index.jsx

```diff
@@ -0,0 +1,155 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'gridicons';
+import StepWrapper from 'signup/step-wrapper';
+import Button from 'components/button';
+import Card from 'components/card';
+import CompactCard from 'components/card/compact';
+import HappychatButton from 'components/happychat/button';
+import RewindCredentialsForm from 'components/rewind-credentials-form';
+import Popover from 'components/popover';
+import SignupActions from 'lib/signup/actions';
+import { getJetpackCredentials, getJetpackCredentialsUpdateStatus } from 'state/selectors';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+class RewindFormStep extends Component {
+	static propTypes = {
+		flowName: PropTypes.string,
+		goToNextStep: PropTypes.func.isRequired,
+		positionInFlow: PropTypes.number,
+		signupProgress: PropTypes.array,
+		stepName: PropTypes.string,
+	};
+
+	state = {
+		showPopover: false,
+	};
+
+	skipStep = () => this.props.goToNextStep();
+
+	onCancel = () => this.props.goToStep( 'rewind-confirm' );
+
+	goToComplete = () => {
+		SignupActions.submitSignupStep( {
+			processingMessage: this.props.translate( 'Setting up your site' ),
+			stepName: this.props.stepName,
+		} );
+
+		this.props.goToNextStep();
+	};
+
+	togglePopover = () => this.setState( { showPopover: ! this.state.showPopover } );
+	storePopoverLink = ref => ( this.popoverLink = ref );
+
+	renderStepContent = () => {
+		const { happychatEvent, siteId, translate } = this.props;
+
+		return (
+			<div className="rewind-form__container">
+				<h3 className="rewind-form__title">{ translate( 'Site Credentials' ) }</h3>
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Add site credentials' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).